build/images/flow-aggregator/Dockerfile

```diff
@@ -1,17 +1,19 @@
-ARG OVS_VERSION
 FROM golang:1.15 as flow-aggregator-build
 
 WORKDIR /antrea
 
 COPY . /antrea
 
-RUN make flow-aggregator
+RUN CGO_ENABLED=0 make flow-aggregator
+
+RUN mkdir /new_tmp
 
 FROM scratch
 
 LABEL maintainer="Antrea <projectantrea-dev@googlegroups.com>"
 LABEL description="The docker image for the flow aggregator"
 
-USER root
+COPY --from=flow-aggregator-build /antrea/bin/flow-aggregator /
+COPY --from=flow-aggregator-build /new_tmp /tmp
```

Comment:
What is the purpose of tmp?

Reply:
It's needed for logs. Added a comment here.

Reply:
sorry could you clarify? which logs?
The flow-aggregator should not emit logs to `/tmp`, everything should go to stdout / stderr

Reply:
Sure, I was adding `/tmp` directory because got this error:

```
 >>> kubectl logs flow-aggregator-78cc9f6c78-5jwpp -n flow-aggregator
 I0331 20:33:40.099906       1 log_file.go:99] Set log file max size to 104857600
 I0331 20:33:40.099906       1 log_file.go:99] Set log file max size to 104857600
 log: exiting because of error: log: cannot create log: open /tmp/flow-aggregator.flow-aggregator-78cc9f6c78-5jwpp.unknownuser.log.INFO.20210331-203340.1: no such file or directory
```

Reply:
According to https://github.com/vmware-tanzu/antrea/blob/bb6cd1016e2d2863847ce13052b951e27a654b14/build/yamls/flow-aggregator/base/flow-aggregator.yml#L125, the logs should go to `/var/log/flowaggregator`, where does the `/tmp` come from?
The intent is to have this directory mounted from the host actually (see YAML for Antrea Controller) which is not currently the case for flow-aggregator.yml. This needs to be addressed as well, there is no need to create the directory in the container image (it will be created dynamically by the volume mount).
The `unknownuser` is not ideal, I think you can solve it by adding `ENV USER root` to the Dockerfile

Reply:
Thanks, it seems the `/tmp` is used by klog. I solved it by adding `--logtostderr` to let logs go to stderr.